src/ralph/lib/mixins/models.py

```diff
@@ -88,8 +89,15 @@ def formfield(self, form_class=TaggitTagField, **kwargs):
         return super().formfield(form_class, **kwargs)
 
 
+class RalphTaggableManager(_TaggableManager):
```

Comment:
Yes, because Django Model manager need have self.name, default TaggitManager has not this variable. 

https://github.com/alex/django-taggit/blob/develop/taggit/managers.py#L111
https://github.com/django/django/blob/1.8.12/django/db/models/manager.py#L71